src/CountDown.elm

```diff
@@ -0,0 +1,155 @@
+module Clock exposing (main)
+
+import Browser
+import Html
+import Html.Attributes exposing (style)
+import Maybe
+import Task
+import Time
+
+
+main : Program () Model Msg
+main =
+    Browser.element
+        { init = init
+        , update = update
+        , view = view
+        , subscriptions = subscriptions
+        }
+
+
+
+{-
+   digital clock
+-}
+
+
+type Msg
+    = Tick Time.Posix
+    | SetTimeZone Time.Zone Time.Posix
+
+
+type alias Model =
+    { time : Time.Posix
+    , zone : Time.Zone
+    }
+
+
+subscriptions : Model -> Sub Msg
+subscriptions _ =
+    Time.every 1000 Tick
+
+
+init : flags -> ( Model, Cmd Msg )
+init _ =
+    ( { time = Time.millisToPosix 0
+      , zone = Time.utc
+      }
+    , Task.perform identity <| Task.map2 SetTimeZone Time.here Time.now
+    )
+
+
+update : Msg -> Model -> ( Model, Cmd msg )
+update msg model =
+    case msg of
+        Tick timePosix ->
+            ( { model | time = timePosix }
+            , Cmd.none
+            )
+
+        SetTimeZone zone posix ->
+            ( { model | time = posix, zone = zone }, Cmd.none )
+
+
+timeLeft model =
+    Time.millisToPosix (4895612 - Time.posixToMillis model.time)
```

Comment:
Where is `4895612` coming from?
If it has some interesting meaning - it should have an informative name - i.e. `kickOffMillis`.